<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironmentFoldersWidget</class>
 <widget class="QWidget" name="EnvironmentFoldersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QToolButton" name="btnOpenIconSetFolderInFileBrowser">
     <property name="toolTip">
      <string>Open in browser</string>
     </property>
     <property name="text">
      <string>Open in browser</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtConfigFolder">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Custom icon sets folder:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnResetDefault">
        <property name="text">
         <string>Remove all custom settings and exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnOpenConfigFolderInBrowser">
     <property name="toolTip">
      <string>Open in browser</string>
     </property>
     <property name="text">
      <string>Open in browser</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Configuration folder:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtIconSetFolder">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Custom theme folder:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtThemeFolder"/>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="btnOpenThemeFolderInFileBrowser">
     <property name="toolTip">
      <string>Open custom theme folder in file browser</string>
     </property>
     <property name="text">
      <string>Open in browser</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtConfigFolder</tabstop>
  <tabstop>btnOpenConfigFolderInBrowser</tabstop>
  <tabstop>txtThemeFolder</tabstop>
  <tabstop>btnOpenThemeFolderInFileBrowser</tabstop>
  <tabstop>txtIconSetFolder</tabstop>
  <tabstop>btnOpenIconSetFolderInFileBrowser</tabstop>
  <tabstop>btnResetDefault</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
